### DISCUSSION TITLE
Welcome to AscEmu Discussions!

### DISCUSSION THREAD

**Discussion by Zyres** [AscEmu/AscEmu#827] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by schnek** [comment]
I saw in conversations on other servers in discord that there is no news from AE since 2018.

this is the last text they see (on the site)
> Welcome 2018
> It seems we forgot to welcome you to 2018! We're glad you made it into the new year, and we hope for some impressive contributions...

news about the project is now here

> http://board.ascemu.org/index.php/Board/32-Core-Script-development/

again, maybe someone doesn't know this. 🐱‍👤

😃it might be worth adding a channel to Discord with links to this information.

**Comment by schnek** [comment]
Maybe need to update goals (projects) ![image](https://user-images.githubusercontent.com/1216225/174435179-11f4fa0d-41a4-4b99-9aeb-30d7bfffb12e.png) now there is a new system.
